shapeshift/web
# fix: unbreak lp opportunity crypto amount

## Description


* fixes crypto amount display for LP opportunities on earn page
## Notice




## Pull Request Type


## Issue (if applicable)



closes #

## Risk


None
## Testing
* Load the application at `app.shapeshift.com` and connect the demo wallet.
* Navigate to the earn page, and view both the fiat and crypto amounts for the FOX-ETH LP opportunity.
* Deploy this branch locally, connect the demo wallet, and navigate to the earn page.
* Verify that both the fiat and crypto amounts for the FOX-ETH LP opportunity appear correct.


### Engineering



### Operations



## Screenshots (if applicable)
* Earn page from production environment with demo wallet connected

<img width="790" alt="Screenshot 2023-01-20 at 8 14 37 PM" src="https://user-images.githubusercontent.com/62026038/213841535-fd3d0432-71d1-4a8e-8c79-bae81f481642.png">


* Earn page from local environment with demo wallet connected
<img width="790" alt="Screenshot 2023-01-20 at 8 14 37 PM" src="https://user-images.githubusercontent.com/62026038/213841537-44df309e-f046-49dd-9004-869b138dbe5f.png">
